incubator/clickhouse/values.yaml

```diff
@@ -0,0 +1,163 @@
+## Pod Annotations
+# podAnnotations: {}
+
+## Statefulset annotations
+# annotations: {}
+
+## Number of ClickHouse nodes
+replicas: 1
+
+## Image settings
+## ref: http://kubernetes.io/docs/user-guide/images/
+image: yandex/clickhouse-server
+imageTag: latest
+imagePullPolicy: Always
+# imagePullSecret: mypullsecret
+
+## Resource limits and requests
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+resources:
+  limits:
+    cpu: 1
+    memory: 1Gi
+  requests:
+    cpu: 1
+    memory: 1Gi
```

Comment:
Per the [review guidelines](https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md):

> It is usually best to not specify defaults for resources and to just provide sensible values that are commented out as a recommendation, especially when resources are rather high. This makes it easier to test charts on small clusters or Minikube. Setting resources should generally be a conscious choice of the user.

Reply:
Made that whole block optional